docs/tutorial-1.md

```diff
@@ -0,0 +1,50 @@
+# How to create an Action
+
+## A first visit
+
+After all steps from the [installation](./installation.md) are running a visit to [http://localhost:8080](http://localhost:8080) should show the following UI:
+
+
+![Cicero WebUI Intro](./cicero_webui_intro.png "Cicero WebUI Intro")
+
+The first visit should also change the url to: [http://localhost:8080/action/current?active](http://localhost:8080/action/current?active)
+
+## Click on "Create New Action"
+After clicking the "Create New Action" button the url should change to:
+[http://localhost:8080/action/new](http://localhost:8080/action/new)
+
+And the UI should correspond to:
+![Cicero WebUI New Action](./cicero_webui_new_action.png "Cicero WebUI New Action")
+
+## Enter the path to the Actions, which should be created
+
+**For any url the 'https://' part needs to be omitted.**
+
+The path to the Actions can be a remote path like [github.com/input-output-hk/cicero](github.com/input-output-hk/cicero)
+or a local path like [/home/developer/IOHK/cicero](/home/developer/IOHK/cicero).
+
+**This doesn't need to be a path to Cicero itself:**
```

Comment:
Are there some requirements for this directory?
I crated empty and when I tried to use it as source I get:
```text
While listing Actions in "/home/parallels/foo": No evaluator succeeded.: Evaluator "nix" failed: Failed to evaluate: exit status 1
Stderr: error: path '/home/parallels/foo' is not a flake (because it doesn't contain a 'flake.nix' file)

Stdout: 
```
in logs I see complains about `flake`:
```text
2022-04-29T14:30:05+02:00 ERR Handler error error="While listing Actions in \"/home/parallels/foo\": No evaluator succeeded.: Evaluator \"nix\" failed: Failed to evaluate: exit status 1\nStderr: error: path '/home/parallels/foo' is not a flake (because it doesn't contain a 'flake.nix' file)\n\nStdout: " component=Web status=500
```

I am at up to date cicero (commit ca625d8798c5ac13d335e64e70eb1434d6eae765) with nix 2.5:
```sh
[cicero]# nix --version
nix (Nix) 2.5.1
```

Reply:
In this example Cicero itself is used, which already has a flake.nix.
In the later chapter the requirements for a repository are better explained: https://cicero-handbook.infra.aws.iohkdev.io/tutorial-3.html